scripts/lib/api/generateApiComponents.ts

```diff
@@ -117,7 +117,7 @@ function prepareProps(
   // Remove the attributes and properties modifiers as we don't show their signatures,
   // but we still use them to create their headers
   if (apiType == "attribute" || apiType == "property") {
-    findByText($, $main, "em.property", apiType).remove();
+    findByText($, $child, "em.property", apiType).remove();
```

Comment:
This change is to restrict the search to the signature node. Before we were searching in the whole DOM which should be slower. Tested regenerating some historical versions (not all of them) as well as qiskit 1.0 and the dev version.